packages/edit-site/src/components/navigation-sidebar/navigation-panel/navigation-item-text.js

```diff
@@ -0,0 +1,7 @@
+export default function NavigationItemText( { title } ) {
+	return (
+		<li className={ 'edit-site-navigation-panel__navigation-item-text' }>
+			{ title }
+		</li>
+	);
+}
```

Comment:
How do you feel if we had this as a proper `Navigation` component?

Let alone the fact that we have a use case here, it could be valuable to export an always-visible item, saving consumers the need to guess/copypaste the `li` styles from `NavigationItem`. 🤔

Reply:
Working on it! See comment above.